File: FrontEnd/SoftPac/SoftPacPersistance/DAO/AcreedoresDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface AcreedoresDAO
    {
        int insertar(AcreedoresDTO acreedor);
        int modificar(AcreedoresDTO acreedor);
        int eliminar(AcreedoresDTO acreedor);
        int eliminarLogico(AcreedoresDTO acreedor);
        AcreedoresDTO obtenerPorId(int acreedorId);
        IList<AcreedoresDTO> ListarTodos();

    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/AcreedoresDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.ArrayList;
import java.util.List;

import pe.edu.pucp.softpac.model.AcreedoresDTO;
import pe.edu.pucp.softpac.model.CuentasAcreedorDTO;
import pe.edu.pucp.softpac.model.UsuariosDTO;

public interface AcreedoresDAO {
    public Integer insertar(AcreedoresDTO acreedor);
    
    public AcreedoresDTO obtenerPorId(Integer acreedorId);
    
    public List<AcreedoresDTO> listarTodos();
    
    public Integer modificar(AcreedoresDTO acreedor);
    
    public Integer eliminar(AcreedoresDTO acreedor);

    public Integer eliminarLogico(AcreedoresDTO acreedor);
}
